### PR TITLE
LPS-148962 | Add baseline images for Clay Sample Portlet

### PR DESCRIPTION
**Pull request by susanchen3**
**Background**
Add baseline images for ocular testing. Also had to remove buttons check because setting the window size made the portlet larger than the window size.

**JIRA Ticket:**
https://issues.liferay.com/browse/LPS-148962

### CODE REVIEW

**Comment by susanchen3** [comment]
ci:test:frontend:nocompile

**Comment by susanchen3** [comment]
Is there a way to check on the image it produces from ci? :thinking:

**Comment by john-co** [comment]
talk to CI/poshi team about this.

**Comment by john-co** [comment]
Hi @susanchen3 , now that QA test writer can adjust the diff sensitivity per call from [latest poshi release](https://liferay.slack.com/archives/CD7939WBE/p1647984343758739), please do some testing to see if reducing the sensitivity to something like 80% or even 70% will work better

References to slack threads from @sergiojimcos ' analysis:
- https://liferay.slack.com/archives/C02B49LBALA/p1639746813000200?thread_ts=1639655900.003800&cid=C02B49LBALA
- https://liferay.slack.com/archives/C02B49LBALA/p1642692382011100?thread_ts=1642692373.011000&cid=C02B49LBALA

**Comment by john-co** [comment]
I let poshi team know that we need the baseline and comparison image on testray. https://issues.liferay.com/browse/POSHI-316 This prob won't be ready until ~2 months from now but we can keep building out our automation and check in on the progress of that.

**Comment by susanchen3** [comment]
ci:test:sf

**Comment by susanchen3** [comment]
ci:test:frontend:nocompile

**Comment by susanchen3** [comment]
Oh just got word from Calum that the snap folder will be renamed to baseline soon. So probably have to wait for that change to be merged before forwarding this. I can update the folder and images when that change happens.

**Comment by susanchen3** [comment]
ci:test:sf

**Comment by susanchen3** [comment]
ci:test:frontend:nocompile

**Comment by john-co** [comment]
Just started reviewing :)

File: portal-web/test/functional/com/liferay/portalweb/macros/clay/ClaySamplePortlet.macro [hunk 1]
```diff
@@ -1,11 +1,34 @@
...
+				locator1 = "ClaySamplePortlet#CLAY_SECTION",
+				value1 = "clay_smoke/can_render_ui/pagination_bars.png",
+				value2 = "70");
+		}
+		else if ("${tab}" == "Progress Bars") {
+			OcularAssertElementImage(
+				locator1 = "ClaySamplePortlet#CLAY_SECTION",
+				value1 = "clay_smoke/can_render_ui/progress_bars.png",
+				value2 = "80");
+		}
```
Comment (john-co) [inline portal-web/test/functional/com/liferay/portalweb/macros/clay/ClaySamplePortlet.macro:25]: You can probably just use [StrintUtil.replace](https://github.com/liferay/liferay-qa-ee/blob/liferay-qa-docs/poshi-docs/references/pages/Methods.markdown#stringutil) to replace spaces with underscore so you can remove the if else statements. Please see link for other references to poshi methods, I know these aren't very public or easy to find...and the docs aren't maintained.

Implementation of more current methods can be found at: https://github.com/liferay/liferay-portal/blob/master/modules/test/poshi/poshi-core/src/main/java/com/liferay/poshi/core/util/StringUtil.java#L681-L701

**Review by john-co — CHANGES_REQUESTED** [review]
Along with the macro edit, would you be able to exclude the portlet's nav tabs from the ocular baseline images? We only want to target the actual clay component contained in the tab. If we depend on extraneous elements such as the nav tab then future events such as adding more nav tabs could cause this test to unnecessarily fail.

**Comment by susanchen3** [comment]
ci:test:sf

**Comment by susanchen3** [comment]
ci:test:frontend:nocompile

**Comment by susanchen3** [comment]
ci:stop

**Comment by susanchen3** [comment]
ci:test:frontend:nocompile

**Comment by susanchen3** [comment]
seems to work without Navigation Bars according to my testing [here](https://github.com/susanchen3/liferay-portal/pull/28#issuecomment-1097196360).. could you try uploading your version of that image? :sweat_smile: @john-co

**Comment by john-co** [comment]
Sorry I couldn't get this working on my side, I'm having issues with SetWindowSize locally. Please check with @mateusralv or other QA to try if uploading their version would work.

**Comment by mateusralv** [comment]
I updated the base images and tested in this [PR](https://github.com/mateusralv/liferay-portal/pull/10) and the test passed 🟢

**Comment by susanchen3** [comment]
ci:test:frontend:nocompile

**Comment by susanchen3** [comment]
woo hoo it passed :tada: 

Thanks for the images cc: @mateusralv :) I had to replace the stickers one because [your images](https://github.com/liferay-frontend/liferay-portal/pull/1994/commits/8591da114faabf40a0ba9b7247f419216c0c5773) aren't loaded in

cc: @john-co

**Comment by john-co** [comment]
Looks good to me! :sunglasses:

**Comment by john-co** [comment]
> I had to replace the stickers one because [your images](https://github.com/liferay-frontend/liferay-portal/pull/1994/commits/8591da114faabf40a0ba9b7247f419216c0c5773) aren't loaded in

I was going to mention too, good job catching it!

**Review by john-co — APPROVED** [review]

**Pull request closed without merging**